registry/storage/driver/testsuites/testsuites.go

```diff
@@ -269,7 +269,7 @@ func (suite *DriverSuite) TestWriteReadStreamsNonUTF8(c *check.C) {
 	suite.writeReadCompareStreams(c, filename, contents)
 }
 
-// TestWriteReadLargeStreams tests that a 5GB file may be written to the storage
```

Comment:
Near as I can tell golang does not do compaction in any of the garbage collection sweeps. Thus making it vulnerable to memory fragmentation when also allocating very large contiguous memory objects.